src/contexts/sendSearchRequest.ts

```diff
@@ -46,8 +50,8 @@ export const sendSearchRequest = async ({
         numResults: 10,
         corpusKey: corpusKeyList,
         context_config: {
-          sentences_before: includeSummary ? 3 : 2,
-          sentences_after: includeSummary ? 3 : 2,
+          sentences_before: includeSummary ? summaryNumSentences : 2,
+          sentences_after: includeSummary ? summaryNumSentences : 2,
```

Comment:
Oh, interesting... so whatever value is specified for `summaryNumSentences` gets multiplied by 2. I just saw that we do clarify this in the README but I wonder if we can find a name that implies it? `summaryNumSentencesAround`? `summaryNumSentencesBeforeAndAfter`? I dunno, those sound weird to me, too. If we can't think of anything better then let's keep it as-is.

Reply:
yes, I agree those are weird. Will leave as such for now and if we find a good name later we can update.